packages/fx-core/resource/package.nls.json

```diff
@@ -442,13 +442,12 @@
   "core.SampleSelect.title": "Start from a sample",
   "core.SampleSelect.placeholder": "Select a sample",
   "core.SampleSelect.buttons.viewSamples": "View samples",
-  "core.question.pluginAvailability.title": "Select Plugin Availability",
-  "core.pluginAvailability.declarativeCopilot": "Declarative Copilot",
-  "core.pluginAvailability.copilotForM365": "Copilot for Microsoft 365",
-  "core.pluginAvailability.declarativeCopilotAndM365": "Both declarative Copilot and Copilot for Microsoft 365",
-  "core.addPlugin.success": "Plugin \"%s\" added to the project successfully.",
-  "core.addAction.success": "Action \"%s\" added to the project successfully.",
-  "core.addActionAndPlugin.success": "Action \"%s\" and plugin \"%s\" added to the project successfully.",
+  "core.addPlugin.success.vsc": "API plugin \"%s\" added to the project successfully.",
+  "core.addPlugin.success": "API plugin \"%s\" added to the project successfully. You can view plugin manifest in \"%s\".",
```

Comment:
API plugin \"%s\" added to the project successfully. View plugin manifest in \"%s\"